aten/src/ATen/native/nested/NestedTensorMath.cpp

```diff
@@ -1125,23 +1138,82 @@ Tensor _reshape_nested(const Tensor& self, IntArrayRef proposed_shape) {
       "for now reshape cannot change the implicit batch dimension");
   std::vector<IntArrayRef> sizes = NestedTensor_get_sizes(self_ptr),
       strides = NestedTensor_get_strides(self_ptr);
-  const std::vector<int64_t>& offsets = self_ptr->get_offsets();
   // reshaping underlying tensor dimensions does not change offset
   // determine reshaped size and stride
   const Tensor& buffer = self_ptr->get_buffer(),
```

Comment:
if view_nested is returning a view it can't use get the storage of the self tensor and then returned a tensor that shares the underlying storage.

Reply:
Will update it according to your pr [#82658](https://github.com/pytorch/pytorch/pull/82658)